Sources/SecureXPC/Server/SecureMessageAcceptor.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SecureMessageAcceptor.swift
+//  SecureXPC
+//
+//  Created by Josh Kaplan on 2022-01-06
+//
+
+import Foundation
+
+/// Accepts messages which meet the provided code signing requirements.
+///
+/// Uses undocumented functionality prior to macOS 11.
+internal struct SecureMessageAcceptor {
```

Comment:
Should this be behind a protocol, so you can make an "always allow" implementation?

Reply:
I originally had it written that way, but it ended up with quite a bit more code across multiple files so it didn't seem worth the complexity.

Reply:
Fair enough. If we need it in the future, we can just do an "extract interface" refactor.

Reply:
Yup, can absolutely refactor in the future if warranted